crates/ruff_workspace/src/options.rs

```diff
@@ -914,6 +915,8 @@ pub struct LintCommonOptions {
             # Ignore `E402` (import violations) in all `__init__.py` files, and in `path/to/file.py`.
             "__init__.py" = ["E402"]
             "path/to/file.py" = ["E402"]
+            # Ignore `D` rules everywhere except for the `src/` directory.
```

Comment:
This comment doesn't make sense to me

Reply:
Ah yeah, it should be `D` below (or the comment should change to `F401`) \cc @carljm

Reply:
Oops, thanks, will fix!